models/consistency_test.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDeleteOrphanedObjects(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+
```

Comment:
This test should add some orphaned objects.

Reply:
Yes just add an pull emtry with repo&issue id = 10000 and we should catch it

Reply:
Since all tests shared the only one fixture directory, I don't think we should change the fixture to an inconsistent state. And also I cannot find any function to create an empty pull request into database.

Reply:
Well we have to exec custom sql. Simple "INSERT INTO x () VALUES(1000,1000);" so i do testing manualy too